Corona-Warn-App/build.gradle

```diff
@@ -37,7 +37,7 @@ android {
         versionName "1.3.0"
 
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
-        
+
```

Comment:
unintended newline?

Reply:
Codeformatter removed the indentation spaces on an empty line.

Someone apparently didn't run the formatter on it when making changes, I did when commiting, and apparently klint/lint doesn't check `build.gradle` as thoroughly as the `.kt` files.